baybe/parameters/enum.py

```diff
@@ -17,17 +17,60 @@ class CategoricalEncoding(ParameterEncoding):
     """Integer encoding."""
 
 
+class SubstanceEncodingAliases(ParameterEncoding):
+    """Aliases for SubstanceEncoding values."""
+
+    MORGAN_FP = "ECFP"
```

Comment:
No, They use different defaults   ` fp_size: int = 2048, radius: int = 2`

I can adapt Enum so that it does not map fp class names to user param names (e.g. now the enum directly maps MORGAN to ECFP). Then the substance parameter class would need to take care of the conversion, which could also include setting appropriate kwargs. This would also enable making encoding field converter leaner as I could use the same enum notation that you use elsewhere where param name = param name in the enum and not class name = param name as I currently have it.

Reply:
hmm im unsure, sounds like the change could be worth it

can you make the changes you propose in a single contained commit? it would be easier to understand everything that needs change then

Reply:
Done in this commit https://github.com/emdgroup/baybe/pull/364/commits/d8903cc171f2502d7c0b7f00e279bfff50cbc54c